lib/models/file.dart

```diff
@@ -275,13 +275,9 @@ class File extends EnteFile {
   }
 
   @override
+  //add nullable return type when migrating to null safety
   String cacheKey() {
     // todo: Neeraj: 19thJuly'22: evaluate and add fileHash as the key?
     return localID ?? uploadedFileID?.toString() ?? generatedID?.toString();
```

Comment:
@ua741 I made `cacheKey()` from `ente_file.dart` have a nullable return type because of this check. Could return null if every variable is null here. Also looks like this method is unused here, you have a `todo` comment on it, so should I keep it or remove it? cacheKey() can be non-nullable if this code is removed.

The return type must be the same type as (or a subtype of) the overridden method’s return type. So cacheKey() from `ente_file.dart` which is the overridden method must have a return type of `String?`